Update BIOMOD_tuning.R

fbreiner opened the pull request:
small updates

fbreiner commented:
Dear Damien,

no idea what is causing this problem because I did not include any warnings in my code.

Could it be caused by the modified tuning.maxent function from the ENMeval package that I am using as an internal function where there is some piece of code like this:


    if (any(is.na(colSums(pres)))) {
      message("Warning: some predictors variables are NA at some occurrence points")
    }
    if (any(is.na(colSums(bg)))) {
      message("Warning: some predictors variables are NA at some background points")
    }


Cheers

Frank

________________________________
Von: Damien Georges <notifications@github.com>
Gesendet: Montag, 12. März 2018 16:03
An: biomodhub/biomod2
Cc: Frank Breiner; Author
Betreff: Re: [biomodhub/biomod2] Update BIOMOD_tuning.R (#3)


@DamienGeorges approved this pull request.

Dear Frank,

Thanks for the patch.
I updated the file but it still fail to pass the current R cran packages procedure which do not permit warnings anymore.

You can have a look to the travis-CI checking procedure

https://travis-ci.org/biomodhub/biomod2/builds/352371068?utm_source=github_status&utm_medium=notification

to see what is causing troubles.

That would be very great if you can fix it one day or another.

Best,
Damien

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/biomodhub/biomod2/pull/3#pullrequestreview-103081288>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AT-mDZM3IKQiOrWrGhTDEuvit1vfwYqIks5tdo5cgaJpZM4SmxvH>.

{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/biomodhub/biomod2","title":"biomodhub/biomod2","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/biomodhub/biomod2"}},"updates":{"snippets":[{"icon":"PERSON","message":"@DamienGeorges approved #3"}],"action":{"name":"View Pull Request","url":"https://github.com/biomodhub/biomod2/pull/3#pullrequestreview-103081288"}}}

DamienGeorges commented:
In fact I was mentioning the following lines in the travis log file (checking package section: l 1291- 1347)

 

checking dependencies in R code ... WARNING

'library' or 'require' call not declared from: ‘PresenceAbsence’

'library' or 'require' calls in package code:

  ‘gam’ ‘PresenceAbsence’

  Please use :: or requireNamespace() instead.

  See section 'Suggested packages' in the 'Writing R Extensions' manual.checking for code/documentation mismatches ... WARNING

Codoc mismatches from documentation object 'BIOMOD_presenceonly':

BIOMOD_presenceonly

  Code: function(modeling.output = NULL, EM.output = NULL, bg.env =

                 NULL, perc = 0.9, save.output = T)

  Docs: function(modeling.output = NULL, EM.output = NULL, save.output

                 = T)

  Argument names in code not in docs:

    bg.env perc

  Mismatches in argument names:

    Position: 3 Code: bg.env Docs: save.output

checking Rd \usage sections ... WARNING

Undocumented arguments in documentation object 'BIOMOD_tuning'

  ‘occ’ ‘env’ ‘pres’ ‘bg’ ‘bg.coords’ ‘occ.grp’ ‘bg.grp’ ‘method’

  ‘maxent.args’ ‘args.lab’ ‘categoricals’ ‘aggregation.factor’ ‘kfolds’

  ‘bin.output’ ‘clamp’ ‘rasterPreds’ ‘parallel’ ‘numCores’ ‘RMvalues’

  ‘fc’

Documented arguments not in \usage in documentation object 'BIOMOD_tuning':

  ‘data’ ‘models’ ‘models.options’ ‘trControl’ ‘ctrl.GAM’ ‘ctrl.GBM’

  ‘ctrl.GLM’ ‘ctrl.CTA’ ‘ctrl.RF’ ‘ctrl.ANN’ ‘ctrl.MARS’ ‘ctrl.FDA’

  ‘metric’ ‘metric.ME’ ‘tuneLength’ ‘method.RF’ ‘method.ANN’

  ‘method.MARS’ ‘method.GAM’ ‘method.GLM’ ‘type.GLM’ ‘interaction.GLM���

  ‘cvmethod.ME’ ‘kfolds.ME’ ‘overlap.ME’ ‘clamp.ME’ ‘n.bg.ME’ ‘env.ME’

  ‘size.tune.ANN’ ‘decay.tune.ANN’ ‘maxit.ANN’ ‘MaxNWts.ANN’

  ‘parallel.ME’ ‘numCores.ME’

Functions with \usage entries need to have the appropriate \alias

entries, and all their arguments documented.

The \usage entries must correspond to syntactically valid R code.

See chapter ‘Writing R documentation files’ in the ‘Writing R

Extensions’ manual.checking for hidden files and directories 

 

That are really small details that should not reauire much work :

 

-          Require/library call of packages to be replaced by requireNamespace()

-          Function argument documentation

-          …

 

Does it make sens?

 

Cheers,

Damien 

 

From: fbreiner [mailto:notifications@github.com] 
Sent: 12 March 2018 16:50
To: biomodhub/biomod2 <biomod2@noreply.github.com>
Cc: Damien Georges <damien.georges2@gmail.com>; Mention <mention@noreply.github.com>
Subject: Re: [biomodhub/biomod2] Update BIOMOD_tuning.R (#3)

 

Dear Damien,

no idea what is causing this problem because I did not include any warnings in my code.

Could it be caused by the modified tuning.maxent function from the ENMeval package that I am using as an internal function where there is some piece of code like this:


if (any(is.na(colSums(pres)))) {
message("Warning: some predictors variables are NA at some occurrence points")
}
if (any(is.na(colSums(bg)))) {
message("Warning: some predictors variables are NA at some background points")
}


Cheers

Frank

________________________________
Von: Damien Georges <notifications@github.com>
Gesendet: Montag, 12. März 2018 16:03
An: biomodhub/biomod2
Cc: Frank Breiner; Author
Betreff: Re: [biomodhub/biomod2] Update BIOMOD_tuning.R (#3)


@DamienGeorges approved this pull request.

Dear Frank,

Thanks for the patch.
I updated the file but it still fail to pass the current R cran packages procedure which do not permit warnings anymore.

You can have a look to the travis-CI checking procedure

https://travis-ci.org/biomodhub/biomod2/builds/352371068?utm_source=github_status&utm_medium=notification

to see what is causing troubles.

That would be very great if you can fix it one day or another.

Best,
Damien

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/biomodhub/biomod2/pull/3#pullrequestreview-103081288>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AT-mDZM3IKQiOrWrGhTDEuvit1vfwYqIks5tdo5cgaJpZM4SmxvH>.

{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/biomodhub/biomod2","title":"biomodhub/biomod2","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/biomodhub/biomod2"}},"updates":{"snippets":[{"icon":"PERSON","message":"@DamienGeorges approved #3"}],"action":{"name":"View Pull Request","url":"https://github.com/biomodhub/biomod2/pull/3#pullrequestreview-103081288"}}}



—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/biomodhub/biomod2/pull/3#issuecomment-372358549> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AJAtuSdhD1XT7Ei-HohBPZutXjfd28p5ks5tdpkogaJpZM4SmxvH> .

fbreiner commented:
Aah, ok now I understand. I will try to solve the issues soon.

Cheers

Frank


________________________________
Von: Damien Georges <notifications@github.com>
Gesendet: Montag, 12. März 2018 17:06
An: biomodhub/biomod2
Cc: Frank Breiner; Author
Betreff: Re: [biomodhub/biomod2] Update BIOMOD_tuning.R (#3)

In fact I was mentioning the following lines in the travis log file (checking package section: l 1291- 1347)



checking dependencies in R code ... WARNING

'library' or 'require' call not declared from: ‘PresenceAbsence’

'library' or 'require' calls in package code:

‘gam’ ‘PresenceAbsence’

Please use :: or requireNamespace() instead.

See section 'Suggested packages' in the 'Writing R Extensions' manual.checking for code/documentation mismatches ... WARNING

Codoc mismatches from documentation object 'BIOMOD_presenceonly':

BIOMOD_presenceonly

Code: function(modeling.output = NULL, EM.output = NULL, bg.env =

NULL, perc = 0.9, save.output = T)

Docs: function(modeling.output = NULL, EM.output = NULL, save.output

= T)

Argument names in code not in docs:

bg.env perc

Mismatches in argument names:

Position: 3 Code: bg.env Docs: save.output

checking Rd \usage sections ... WARNING

Undocumented arguments in documentation object 'BIOMOD_tuning'

‘occ’ ‘env’ ‘pres’ ‘bg’ ‘bg.coords’ ‘occ.grp’ ‘bg.grp’ ‘method’

‘maxent.args’ ‘args.lab’ ‘categoricals’ ‘aggregation.factor’ ‘kfolds’

‘bin.output’ ‘clamp’ ‘rasterPreds’ ‘parallel’ ‘numCores’ ‘RMvalues’

‘fc’

Documented arguments not in \usage in documentation object 'BIOMOD_tuning':

‘data’ ‘models’ ‘models.options’ ‘trControl’ ‘ctrl.GAM’ ‘ctrl.GBM’

‘ctrl.GLM’ ‘ctrl.CTA’ ‘ctrl.RF’ ‘ctrl.ANN’ ‘ctrl.MARS’ ‘ctrl.FDA’

‘metric’ ‘metric.ME’ ‘tuneLength’ ‘method.RF’ ‘method.ANN’

‘method.MARS’ ‘method.GAM’ ‘method.GLM’ ‘type.GLM’ ‘interaction.GLM���

‘cvmethod.ME’ ‘kfolds.ME’ ‘overlap.ME’ ‘clamp.ME’ ‘n.bg.ME’ ‘env.ME’

‘size.tune.ANN’ ‘decay.tune.ANN’ ‘maxit.ANN’ ‘MaxNWts.ANN’

‘parallel.ME’ ‘numCores.ME’

Functions with \usage entries need to have the appropriate \alias

entries, and all their arguments documented.

The \usage entries must correspond to syntactically valid R code.

See chapter ‘Writing R documentation files’ in the ‘Writing R

Extensions’ manual.checking for hidden files and directories



That are really small details that should not reauire much work :



- Require/library call of packages to be replaced by requireNamespace()

- Function argument documentation

- …



Does it make sens?



Cheers,

Damien



From: fbreiner [mailto:notifications@github.com]
Sent: 12 March 2018 16:50
To: biomodhub/biomod2 <biomod2@noreply.github.com>
Cc: Damien Georges <damien.georges2@gmail.com>; Mention <mention@noreply.github.com>
Subject: Re: [biomodhub/biomod2] Update BIOMOD_tuning.R (#3)



Dear Damien,

no idea what is causing this problem because I did not include any warnings in my code.

Could it be caused by the modified tuning.maxent function from the ENMeval package that I am using as an internal function where there is some piece of code like this:


if (any(is.na(colSums(pres)))) {
message("Warning: some predictors variables are NA at some occurrence points")
}
if (any(is.na(colSums(bg)))) {
message("Warning: some predictors variables are NA at some background points")
}


Cheers

Frank

________________________________
Von: Damien Georges <notifications@github.com>
Gesendet: Montag, 12. März 2018 16:03
An: biomodhub/biomod2
Cc: Frank Breiner; Author
Betreff: Re: [biomodhub/biomod2] Update BIOMOD_tuning.R (#3)


@DamienGeorges approved this pull request.

Dear Frank,

Thanks for the patch.
I updated the file but it still fail to pass the current R cran packages procedure which do not permit warnings anymore.

You can have a look to the travis-CI checking procedure

https://travis-ci.org/biomodhub/biomod2/builds/352371068?utm_source=github_status&utm_medium=notification

to see what is causing troubles.

That would be very great if you can fix it one day or another.

Best,
Damien

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/biomodhub/biomod2/pull/3#pullrequestreview-103081288>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AT-mDZM3IKQiOrWrGhTDEuvit1vfwYqIks5tdo5cgaJpZM4SmxvH>.

{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/biomodhub/biomod2","title":"biomodhub/biomod2","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/biomodhub/biomod2"}},"updates":{"snippets":[{"icon":"PERSON","message":"@DamienGeorges approved #3"}],"action":{"name":"View Pull Request","url":"https://github.com/biomodhub/biomod2/pull/3#pullrequestreview-103081288"}}}



—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/biomodhub/biomod2/pull/3#issuecomment-372358549> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AJAtuSdhD1XT7Ei-HohBPZutXjfd28p5ks5tdpkogaJpZM4SmxvH> .


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/biomodhub/biomod2/pull/3#issuecomment-372364372>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AT-mDQPpI3ZEiTBeDQ1qAP8HICCKEX-3ks5tdpzqgaJpZM4SmxvH>.

{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/biomodhub/biomod2","title":"biomodhub/biomod2","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/biomodhub/biomod2"}},"updates":{"snippets":[{"icon":"PERSON","message":"@DamienGeorges in #3: In fact I was mentioning the following lines in the travis log file (checking package section: l 1291- 1347)\n\n \n\nchecking dependencies in R code ... WARNING\n\n'library' or 'require' call not declared from: ‘PresenceAbsence’\n\n'library' or 'require' calls in package code:\n\n ‘gam’ ‘PresenceAbsence’\n\n Please use :: or requireNamespace() instead.\n\n See section 'Suggested packages' in the 'Writing R Extensions' manual.checking for code/documentation mismatches ... WARNING\n\nCodoc mismatches from documentation object 'BIOMOD_presenceonly':\n\nBIOMOD_presenceonly\n\n Code: function(modeling.output = NULL, EM.output = NULL, bg.env =\n\n NULL, perc = 0.9, save.output = T)\n\n Docs: function(modeling.output = NULL, EM.output = NULL, save.output\n\n = T)\n\n Argument names in code not in docs:\n\n bg.env perc\n\n Mismatches in argument names:\n\n Position: 3 Code: bg.env Docs: save.output\n\nchecking Rd \\usage sections ... WARNING\n\nUndocumented arguments in documentation object 'BIOMOD_tuning'\n\n ‘occ’ ‘env’ ‘pres’ ‘bg’ ‘bg.coords’ ‘occ.grp’ ‘bg.grp’ ‘method’\n\n ‘maxent.args’ ‘args.lab’ ‘categoricals’ ‘aggregation.factor’ ‘kfolds’\n\n ‘bin.output’ ‘clamp’ ‘rasterPreds’ ‘parallel’ ‘numCores’ ‘RMvalues’\n\n ‘fc’\n\nDocumented arguments not in \\usage in documentation object 'BIOMOD_tuning':\n\n ‘data’ ‘models’ ‘models.options’ ‘trControl’ ‘ctrl.GAM’ ‘ctrl.GBM’\n\n ‘ctrl.GLM’ ‘ctrl.CTA’ ‘ctrl.RF’ ‘ctrl.ANN’ ‘ctrl.MARS’ ‘ctrl.FDA’\n\n ‘metric’ ‘metric.ME’ ‘tuneLength’ ‘method.RF’ ‘method.ANN’\n\n ‘method.MARS’ ‘method.GAM’ ‘method.GLM’ ‘type.GLM’ ‘interaction.GLM���\n\n ‘cvmethod.ME’ ‘kfolds.ME’ ‘overlap.ME’ ‘clamp.ME’ ‘n.bg.ME’ ‘env.ME’\n\n ‘size.tune.ANN’ ‘decay.tune.ANN’ ‘maxit.ANN’ ‘MaxNWts.ANN’\n\n ‘parallel.ME’ ‘numCores.ME’\n\nFunctions with \\usage entries need to have the appropriate \\alias\n\nentries, and all their arguments documented.\n\nThe \\usage entries must correspond to syntactically valid R code.\n\nSee chapter ‘Writing R documentation files’ in the ‘Writing R\n\nExtensions’ manual.checking for hidden files and directories \n\n \n\nThat are really small details that should not reauire much work :\n\n \n\n- Require/library call of packages to be replaced by requireNamespace()\n\n- Function argument documentation\n\n- …\n\n \n\nDoes it make sens?\n\n \n\nCheers,\n\nDamien \n\n \n\nFrom: fbreiner [mailto:notifications@github.com] \nSent: 12 March 2018 16:50\nTo: biomodhub/biomod2 \u003cbiomod2@noreply.github.com\u003e\nCc: Damien Georges \u003cdamien.georges2@gmail.com\u003e; Mention \u003cmention@noreply.github.com\u003e\nSubject: Re: [biomodhub/biomod2] Update BIOMOD_tuning.R (#3)\n\n \n\nDear Damien,\n\nno idea what is causing this problem because I did not include any warnings in my code.\n\nCould it be caused by the modified tuning.maxent function from the ENMeval package that I am using as an internal function where there is some piece of code like this:\n\n\nif (any(is.na(colSums(pres)))) {\nmessage(\"Warning: some predictors variables are NA at some occurrence points\")\n}\nif (any(is.na(colSums(bg)))) {\nmessage(\"Warning: some predictors variables are NA at some background points\")\n}\n\n\nCheers\n\nFrank\n\n________________________________\nVon: Damien Georges \u003cnotifications@github.com\u003e\nGesendet: Montag, 12. März 2018 16:03\nAn: biomodhub/biomod2\nCc: Frank Breiner; Author\nBetreff: Re: [biomodhub/biomod2] Update BIOMOD_tuning.R (#3)\n\n\n@DamienGeorges approved this pull request.\n\nDear Frank,\n\nThanks for the patch.\nI updated the file but it still fail to pass the current R cran packages procedure which do not permit warnings anymore.\n\nYou can have a look to the travis-CI checking procedure\n\nhttps://travis-ci.org/biomodhub/biomod2/builds/352371068?utm_source=github_status\u0026utm_medium=notification\n\nto see what is causing troubles.\n\nThat would be very great if you can fix it one day or another.\n\nBest,\nDamien\n\n-\nYou are receiving this because you authored the thread.\nReply to this email directly, view it on GitHub\u003chttps://github.com/biomodhub/biomod2/pull/3#pullrequestreview-103081288\u003e, or mute the thread\u003chttps://github.com/notifications/unsubscribe-auth/AT-mDZM3IKQiOrWrGhTDEuvit1vfwYqIks5tdo5cgaJpZM4SmxvH\u003e.\n\n{\"api_version\":\"1.0\",\"publisher\":{\"api_key\":\"05dde50f1d1a384dd78767c55493e4bb\",\"name\":\"GitHub\"},\"entity\":{\"external_key\":\"github/biomodhub/biomod2\",\"title\":\"biomodhub/biomod2\",\"subtitle\":\"GitHub repository\",\"main_image_url\":\"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png\",\"avatar_image_url\":\"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png\",\"action\":{\"name\":\"Open in GitHub\",\"url\":\"https://github.com/biomodhub/biomod2\"}},\"updates\":{\"snippets\":[{\"icon\":\"PERSON\",\"message\":\"@DamienGeorges approved #3\"}],\"action\":{\"name\":\"View Pull Request\",\"url\":\"https://github.com/biomodhub/biomod2/pull/3#pullrequestreview-103081288\"}}}\n\n\n\n—\nYou are receiving this because you were mentioned.\nReply to this email directly, view it on GitHub \u003chttps://github.com/biomodhub/biomod2/pull/3#issuecomment-372358549\u003e , or mute the thread \u003chttps://github.com/notifications/unsubscribe-auth/AJAtuSdhD1XT7Ei-HohBPZutXjfd28p5ks5tdpkogaJpZM4SmxvH\u003e .\n\n"}],"action":{"name":"View Pull Request","url":"https://github.com/biomodhub/biomod2/pull/3#issuecomment-372364372"}}}

fbreiner commented:
Dear Damien,


I had a look for the errors reported from CRAN. There is one problem with the tuning.maxent function which I have modified from the ENMeval package. This is an internal function for the BIOMOD_tuning function and CRAN is now asking to define the arguments of this internal function in the documentation object. Is there any way to define this as internal function without the need of defining all the arguments? The argumentts pf this function should not be changed by the user.

I also saw that in the  'BIOMOD_tuning.Rd' file the function in \usage{} is the internal tuning.maxent function and not the  BIOMOD_tuning' function. This seems to be an error. Should I simply change that in the .Rd file?


Cheers

Frank


________________________________
Von: Damien Georges <notifications@github.com>
Gesendet: Montag, 12. März 2018 16:03
An: biomodhub/biomod2
Cc: Frank Breiner; Author
Betreff: Re: [biomodhub/biomod2] Update BIOMOD_tuning.R (#3)


@DamienGeorges approved this pull request.

Dear Frank,

Thanks for the patch.
I updated the file but it still fail to pass the current R cran packages procedure which do not permit warnings anymore.

You can have a look to the travis-CI checking procedure

https://travis-ci.org/biomodhub/biomod2/builds/352371068?utm_source=github_status&utm_medium=notification

to see what is causing troubles.

That would be very great if you can fix it one day or another.

Best,
Damien

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/biomodhub/biomod2/pull/3#pullrequestreview-103081288>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AT-mDZM3IKQiOrWrGhTDEuvit1vfwYqIks5tdo5cgaJpZM4SmxvH>.

{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/biomodhub/biomod2","title":"biomodhub/biomod2","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/biomodhub/biomod2"}},"updates":{"snippets":[{"icon":"PERSON","message":"@DamienGeorges approved #3"}],"action":{"name":"View Pull Request","url":"https://github.com/biomodhub/biomod2/pull/3#pullrequestreview-103081288"}}}

DamienGeorges commented:
Dear Frank,

 

Many thanks for your time on this!

I think that to make a function internal you have togivr it a  name starting with a dot (e.g. .myInternalFunction())

For the .Rd file changing. Please give it a try.

 

Cheers,

Damien

 

From: fbreiner [mailto:notifications@github.com] 
Sent: 21 March 2018 15:11
To: biomodhub/biomod2 <biomod2@noreply.github.com>
Cc: Damien Georges <damien.georges2@gmail.com>; Mention <mention@noreply.github.com>
Subject: Re: [biomodhub/biomod2] Update BIOMOD_tuning.R (#3)

 

Dear Damien,


I had a look for the errors reported from CRAN. There is one problem with the tuning.maxent function which I have modified from the ENMeval package. This is an internal function for the BIOMOD_tuning function and CRAN is now asking to define the arguments of this internal function in the documentation object. Is there any way to define this as internal function without the need of defining all the arguments? The argumentts pf this function should not be changed by the user.

I also saw that in the 'BIOMOD_tuning.Rd' file the function in \usage{} is the internal tuning.maxent function and not the BIOMOD_tuning' function. This seems to be an error. Should I simply change that in the .Rd file?


Cheers

Frank


________________________________
Von: Damien Georges <notifications@github.com <mailto:notifications@github.com> >
Gesendet: Montag, 12. März 2018 16:03
An: biomodhub/biomod2
Cc: Frank Breiner; Author
Betreff: Re: [biomodhub/biomod2] Update BIOMOD_tuning.R (#3)


@DamienGeorges approved this pull request.

Dear Frank,

Thanks for the patch.
I updated the file but it still fail to pass the current R cran packages procedure which do not permit warnings anymore.

You can have a look to the travis-CI checking procedure

https://travis-ci.org/biomodhub/biomod2/builds/352371068?utm_source=github_status <https://travis-ci.org/biomodhub/biomod2/builds/352371068?utm_source=github_status&utm_medium=notification> &utm_medium=notification

to see what is causing troubles.

That would be very great if you can fix it one day or another.

Best,
Damien

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/biomodhub/biomod2/pull/3#pullrequestreview-103081288>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AT-mDZM3IKQiOrWrGhTDEuvit1vfwYqIks5tdo5cgaJpZM4SmxvH>.

{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/biomodhub/biomod2","title":"biomodhub/biomod2","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/biomodhub/biomod2"}},"updates":{"snippets":[{"icon":"PERSON","message":"@DamienGeorges approved #3"}],"action":{"name":"View Pull Request","url":"https://github.com/biomodhub/biomod2/pull/3#pullrequestreview-103081288"}}}



—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/biomodhub/biomod2/pull/3#issuecomment-374950094> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AJAtuYF8rRHF4E7TmtzLIkchxFWLtKt4ks5tgl9ugaJpZM4SmxvH> .  <https://github.com/notifications/beacon/AJAtuQ2GQVDJbwClXHl8YjHxeUcvpMfeks5tgl9ugaJpZM4SmxvH.gif>
